src/plugins/unified_histogram/public/chart/histogram.tsx

```diff
@@ -212,6 +212,7 @@ export function Histogram({
       >
         <lens.EmbeddableComponent
           {...lensProps}
+          forceDSL={true}
```

Comment:
Can you add a small comment about why we need to force the DSL in unified histogram?
Thanks